api/app/tests/test_filtering.py

```diff
@@ -0,0 +1,154 @@
+    def test_match_nothing(self):
+        # Determine boundingbox that contains no Signals (see setUp).
+        min_lon = IN_AMSTERDAM[0] - 0.5 * self.dlon
+        max_lon = IN_AMSTERDAM[0] + 0.5 * self.dlon
+
+        min_lat = IN_AMSTERDAM[1] + (N_RECORDS - 0.5) * self.dlat
+        max_lat = IN_AMSTERDAM[1] + (N_RECORDS + 0.5) * self.dlat
+
+        response = self._get_response(
+            LOCATION_ENDPOINT, {'in_bbox': f'{min_lon},{min_lat},{max_lon},{max_lat}'})
+
+        self.assertEquals(response.status_code, 200)
+        self.assertEquals(response.json()['count'], 0)
+
+    def test_match_everything(self):
+        # Determine boundingbox that contains all Signals (see setUp).
+        min_lon = IN_AMSTERDAM[0] - 0.5 * self.dlon
+        max_lon = IN_AMSTERDAM[0] + (N_RECORDS + 0.5) * self.dlon
+
+        min_lat = IN_AMSTERDAM[1] - 0.5 * self.dlat
+        max_lat = IN_AMSTERDAM[1] + (N_RECORDS + 0.5) * self.dlat
+
+        response = self._get_response(
+            LOCATION_ENDPOINT, {'in_bbox': f'{min_lon},{min_lat},{max_lon},{max_lat}'})
+
+        self.assertEquals(response.status_code, 200)
+        self.assertEquals(response.json()['count'], N_RECORDS)
+
+    def test_match_one(self):
+        # Determine boundingbox that contains one single Signal (see setUp).
+        min_lon = IN_AMSTERDAM[0] - 0.5 * self.dlon
+        max_lon = IN_AMSTERDAM[0] + 0.5 * self.dlon
+
+        min_lat = IN_AMSTERDAM[1] - 0.5 * self.dlat
+        max_lat = IN_AMSTERDAM[1] + 0.5 * self.dlat
+
+        response = self._get_response(
+            LOCATION_ENDPOINT, {'in_bbox': f'{min_lon},{min_lat},{max_lon},{max_lat}'})
+
+        self.assertEquals(response.status_code, 200)
+        self.assertEquals(response.json()['count'], 1)
+
+    def test_filter_signal_via_location(self):
+        # Determine boundingbox that contains one single Signal (see setUp).
+        min_lon = IN_AMSTERDAM[0] - 0.5 * self.dlon
+        max_lon = IN_AMSTERDAM[0] + 0.5 * self.dlon
+
+        min_lat = IN_AMSTERDAM[1] - 0.5 * self.dlat
+        max_lat = IN_AMSTERDAM[1] + 0.5 * self.dlat
+
+        response = self._get_response(
+            SIGNAL_ENDPOINT, {'in_bbox': f'{min_lon},{min_lat},{max_lon},{max_lat}'})
+
+        self.assertEquals(response.status_code, 200)
+        self.assertEquals(response.json()['count'], 1)
+
+    def test_filter_status_via_location(self):
+        # Determine boundingbox that contains one single Status (see setUp).
+        min_lon = IN_AMSTERDAM[0] - 0.5 * self.dlon
+        max_lon = IN_AMSTERDAM[0] + 0.5 * self.dlon
+
+        min_lat = IN_AMSTERDAM[1] - 0.5 * self.dlat
+        max_lat = IN_AMSTERDAM[1] + 0.5 * self.dlat
+
+        response = self._get_response(
+            STATUS_ENDPOINT, {'in_bbox': f'{min_lon},{min_lat},{max_lon},{max_lat}'})
+
+        self.assertEquals(response.status_code, 200)
+        self.assertEquals(response.json()['count'], 1)
+
+
+class TestLocatieFilter(FilterTestDataMixin, APITestCase):
```

Comment:
Typo `Locatie` -> `Location`

Reply:
is een inconsistentie in de API, kan het gaan renamen, maar dan wordt de API ook anders